tests/camel-itest-karaf/src/test/java/org/apache/camel/itest/karaf/CamelRestSwaggerTest.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.camel.itest.karaf;
```

Comment:
please add the header.
for instance https://github.com/apache/camel/blob/eccea3cdb0d357dd655b10b8c220b3fc9f8afa0b/tests/camel-itest-karaf/src/test/java/org/apache/camel/itest/karaf/BaseKarafTest.java#L1-L16